src/main/scala/sangria/marshalling/circe.scala

```diff
@@ -25,8 +25,8 @@ object circe {
       case v: Boolean ⇒ Json.fromBoolean(v)
       case v: Int ⇒ Json.fromInt(v)
       case v: Long ⇒ Json.fromLong(v)
-      case v: Float ⇒ Json.fromDouble(v).get
-      case v: Double ⇒ Json.fromDouble(v).get
+      case v: Float ⇒ Json.fromDoubleOrNull(v)
+      case v: Double ⇒ Json.fromDoubleOrNull(v)
```

Comment:
Even if we merge https://github.com/sangria-graphql/sangria/pull/167, I think we can use `fromDoubleOrNull` here anyway. Main library will ensure that NaN and infinity will not reach this point, but at least we can avoid boxing (which `fromDouble` does).